### PR TITLE
config fix

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/configs/kvk.json [hunk 1]
```diff
@@ -87,27 +87,24 @@
       }
     },
     "SSICredentialVerifyRequestPage": {
-      "enableRightPaneButton" : false,
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/src/configs/kvk.json:90]: This has to stay, we do not need a button on this page

File: packages/oid4vci-demo-frontend/src/configs/kvk.json [hunk 2]
```diff
@@ -87,27 +87,24 @@
...
+      "photoRight": "undraw_messenger_re_8bky_1.svg",
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/src/configs/kvk.json]: I do not see this image in Figma

File: packages/oid4vci-demo-frontend/src/configs/kvk.json [hunk 3]
```diff
@@ -164,14 +148,6 @@
         "width": 383,
         "height": 383
       },
-      "mobile": {
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/src/configs/kvk.json:167]: Sure SSIInformationRequestPage does not need a mobile scetion?

File: packages/oid4vci-demo-frontend/src/configs/kvk.json (hunk 1 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/kvk.json:90]: we are checking for undefined

File: packages/oid4vci-demo-frontend/src/configs/kvk.json (hunk 3 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/kvk.json:167]: design has no logo here

File: packages/oid4vci-demo-frontend/src/configs/kvk.json (hunk 2 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/kvk.json]: https://www.figma.com/file/T5ZNMVDMWVaESbGAqc6Hek/KNB-usecase-of-the-Sphereon-Wallet?type=design&node-id=46%3A7802&mode=dev

has it

File: packages/oid4vci-demo-frontend/src/configs/kvk.json (hunk 3 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/configs/kvk.json:167]: actually

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/kvk.json:167]: ye i got confussed, it has been moved in the config it seems, under the form

**Pull request merged**